3.5/foxx-reference-modules-graph-ql.md

```diff
@@ -54,39 +77,9 @@ router.use('/graphql', createGraphQLRouter({
 }));
 ```
 
-**Note**: ArangoDB aims for stability which means bundled dependencies will
-generally not be updated as quickly as their maintainers make updates
-available on GitHub or NPM. Starting with ArangoDB 3.2, if you want to use a
-newer version than the one bundled with your target version of ArangoDB, you
-can provide your own version of the library by passing it via the `graphql` option:
-
-```js
-const graphql = require('graphql-sync');
-const graphqlSchema = new graphql.Schema({
-  //...
-});
-module.context.use(createGraphQLRouter({
-  schema: graphqlSchema,
-  graphiql: true,
-  graphql: graphql
-}))
-```
-
-Starting with `graphql` 0.12 you can also use
-[the official graphql library](https://github.com/graphql/graphql-js){:target="_blank"} if you
-include it in the `node_modules` folder of your service bundle:
-
-```js
-const graphql = require('graphql'); // 0.12 or later
-const graphqlSchema = new graphql.Schema({
-  //...
-});
-module.context.use(createGraphQLRouter({
-  schema: graphqlSchema,
-  graphiql: true,
-  graphql: graphql
-}))
-```
+For more information on `graphql-sync` see the
+[`graphql-js` API reference](http://graphql.org/docs/api-reference-graphql/){:target="_blank"}
+(note that `graphql-sync` never wraps results in promises).
```

Comment:
also this aside about promises is irrelevant as graphql no longer wraps results in promises unless the resolution method is defined as async, so this just works now. it doesn't do any harm but it's also not informative.